tests/integration/api_v3/http/blockchainAppsMetaList.test.js

```diff
@@ -0,0 +1,98 @@
+
+const {
+	badRequestSchema,
+	goodRequestSchema,
+	metaSchema,
+} = require('../../../schemas/httpGenerics.schema');
+
+const {
+	blockchainAppMetaListSchema,
+} = require('../../../schemas/api_v3/blockchainAppsMetaListSchema.schema');
+
+const baseUrl = config.SERVICE_ENDPOINT;
+const baseUrlV3 = `${baseUrl}/api/v3`;
+const endpoint = `${baseUrlV3}/blockchain/apps/meta/list`;
+
+// TODO: Enable test cases once off-chain data is available
+xdescribe('Blockchain application meta list API', () => {
+	it('retrieves list', async () => {
+		const response = await api.get(endpoint);
+		expect(response).toMap(goodRequestSchema);
+		expect(response.data).toBeInstanceOf(Array);
+		expect(response.data.length).toBeGreaterThanOrEqual(1);
+		expect(response.data.length).toBeLessThanOrEqual(10);
+		response.data.map(blockchainApp => expect(blockchainApp).toMap(blockchainAppMetaListSchema));
+		expect(response.meta).toMap(metaSchema);
+	});
+
+	it('retrieves list with limit=10', async () => {
+		const response = await api.get(`${endpoint}?limit=10`);
+		expect(response).toMap(goodRequestSchema);
+		expect(response.data).toBeInstanceOf(Array);
+		expect(response.data.length).toBeGreaterThanOrEqual(1);
+		expect(response.data.length).toBeLessThanOrEqual(10);
+		response.data.map(blockchainApp => expect(blockchainApp).toMap(blockchainAppMetaListSchema));
+		expect(response.meta).toMap(metaSchema);
+	});
+
+	it('retrieves list with limit=10 and offset=1', async () => {
+		const response = await api.get(`${endpoint}?limit=10&offset=1`);
+		expect(response).toMap(goodRequestSchema);
+		expect(response.data).toBeInstanceOf(Array);
+		expect(response.data.length).toBeGreaterThanOrEqual(1);
+		expect(response.data.length).toBeLessThanOrEqual(10);
+		response.data.map(blockchainApp => expect(blockchainApp).toMap(blockchainAppMetaListSchema));
+		expect(response.meta).toMap(metaSchema);
+	});
+
+	it('retrieves list with limit=10, offset=1 and sort=name:desc', async () => {
+		const response = await api.get(`${endpoint}?limit=10&offset=1&sort=name:desc`);
+		expect(response).toMap(goodRequestSchema);
+		expect(response.data).toBeInstanceOf(Array);
+		expect(response.data.length).toBeGreaterThanOrEqual(1);
+		expect(response.data.length).toBeLessThanOrEqual(10);
+		response.data.map(blockchainApp => expect(blockchainApp).toMap(blockchainAppMetaListSchema));
+		expect(response.meta).toMap(metaSchema);
+	});
+
+	it('retrieves blockchain application meta list by name', async () => {
+		const response = await api.get(`${endpoint}?name=Lisk`);
+		expect(response).toMap(goodRequestSchema);
+		expect(response.data).toBeInstanceOf(Array);
+		expect(response.data.length).toEqual(1);
+		response.data.map(blockchainApp => expect(blockchainApp).toMap(blockchainAppMetaListSchema));
+		expect(response.meta).toMap(metaSchema);
+	});
+
+	it('retrieves blockchain applications meta list by partial search', async () => {
+		const response = await api.get(`${endpoint}?search=Lisk`);
```

Comment:
Please add additional test cases to include partial keyword search and case-insensitive searches.
Please do the same for RPC tests as well.